packages/engine/src/HedronEngine/HedronEngine.ts

```diff
@@ -1,11 +1,12 @@
+import { Midi, MIDIEvent } from '@hedron/midi'
```

Comment:
for some reason this isn't recognised in my vscode, despite being a dependency for engine, maybe it needs to be bumped up to the root?